flink-runtime/src/test/java/org/apache/flink/runtime/webmonitor/threadinfo/JobVertexThreadInfoTrackerTest.java

```diff
@@ -139,7 +139,7 @@ public void testCachedStatsNotUpdatedWithinRefreshInterval() throws Exception {
     /** Tests that cached result is NOT reused after refresh interval. */
     @Test
     public void testCachedStatsUpdatedAfterRefreshInterval() throws Exception {
```

Comment:
This test still seems flaky on my box.

Reply:
Looking into this. I get crazy failure rates when using IntelliJ's repeat count, but all passes when repeating in other ways (e.g. simple `for` loop in the test). Not quite sure what's going on yet...

Reply:
Should be fixed now. I don't quite understand though why this happened only with IntelliJ repetitions, since it seemed an actual issue that should happen fairly reliably...